pom.xml

```diff
@@ -247,6 +247,101 @@
         <quarkus.native.native-image-xmx>6G</quarkus.native.native-image-xmx>
       </properties>
     </profile>
+    <profile>
+      <!-- This profile needs to be activated while working with offline mode related files such as FileWriterSession.java -->
+      <id>offline</id>
+      <repositories>
+        <repository>
+          <id>artifactory</id>
+          <url>https://repo.datastax.com/dse</url>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>true</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+      <dependencies>
+        <dependency>
+          <groupId>com.datastax.dse</groupId>
+          <artifactId>dse-db-all</artifactId>
+          <version>4.0.11-591d171ac9c9</version>
```

Comment:
We have this version referenced from:

```
<stargate.int-test.cassandra.image-tag>
```

but with different actual value -- should probably create a common property to reference (add in main `<properties>` so accessible to profile section too, I think)?

Reply:
Fixed in this commit [4167d29](https://github.com/stargate/data-api/pull/905/commits/4167d29fe3f3d950025e65d9b06eab947a619088)